File: branches/Medusa/Source/Fantasy.Jobs/ITagValueProvider.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Win32;

namespace Fantasy.Jobs
{
    public interface ITagValueProvider
    {
        char Prefix { get; }
        string GetTagValue(string tag, IDictionary<string, object> context);
        bool HasTag(string tag, IDictionary<string, object> context);
        bool IsEnabled(IDictionary<string, object> context);
    }


   

   



}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/ITagValueProvider.java
package Fantasy.Jobs;

import Microsoft.Win32.*;

public interface ITagValueProvider
{
	char getPrefix();
	String GetTagValue(String tag, java.util.Map<String, Object> context);
	boolean HasTag(String tag, java.util.Map<String, Object> context);
	boolean IsEnabled(java.util.Map<String, Object> context);
}